crates/breez-sdk/core/src/models/mod.rs

```diff
@@ -43,6 +44,45 @@ pub struct ConnectRequest {
     pub storage_dir: String,
 }
 
+#[derive(Clone)]
+#[cfg_attr(feature = "uniffi", derive(uniffi::Record))]
+pub struct DefaultStorageRequest {
+    pub storage_dir: String,
+    pub network: Network,
+    pub seed: Seed,
```

Comment:
Not that it makes a world of difference, but it feels a bit odd to pass the seed when creating the storage. We could instead provide just the hash suffix. The logic for getting the hash could be moved to a `impl Seed` block. Wdyt?